doc/sphinx/language/core/index.rst

```diff
@@ -30,10 +30,67 @@ will have to check their output.
 .. toctree::
    :maxdepth: 1
 
+   .. The two chapters Gallina specification language and CIC are
+      planned for removal.  Eventually, the present "Core language"
+      chapter will play the role of the former Gallina chapter, except
+      that some core notions will have moved there from the former
+      Gallina extensions chapter.
+
    ../gallina-specification-language
    ../cic
+
+   .. Comments are used to show upcoming sections (not yet split out).
+
+   .. Lexical conventions, how to read the grammars in the manual,
+      presentations of the basic notions: terms, commands, tactics,
+      attributes and options.
+
+   .. Terms, typing and conversion (only basic constructs:
+      abstraction, product, application, type cast, brief mention of
+      sorts).
+
+   .. Assumptions
+
+   .. Definitions (vernac) and local definitions (let in).  Includes
+      opaque definitions and a brief introduction to the interactive
+      proof mode, in the spirit of CEP #42.
+
+   .. Variants and the match-construct
+
    records
-   ../../addendum/universe-polymorphism
+
+   .. Inductive types and recursive definitions (Inductive, fix and
+      Fixpoint)
+
+   .. Co-inductive types and co-recursive definitions (CoInductive,
+      cofix and CoFixpoint)
+
+   .. Conversion (including fast reduction machines)
+
+   .. Native integers and floats (and arrays when they arrive).  Here
+      or in the Libraries chapter or merged with the section on
+      conversion above?
+
+   .. The next section would be about sorts, would include the current
+      addendum chapter about SProp, but also explain clearly
+      Set and Prop, imprecativity, the -imprecative-set option.
+
    ../../addendum/sprop
+
+   .. The next section would be about universes, including how to
```

Comment:
"would be" makes it sound like the section doesn't exist.  The comment describes planned changes/additions, right?  Maybe "will add"?

Same comment on 2 other "would be"s, one above and below